src/api/list_add.sql

```diff
@@ -0,0 +1,60 @@
+DROP FUNCTION IF EXISTS api.list_add(jsonb);
+
+CREATE OR REPLACE FUNCTION api.list_add(data jsonb)
+RETURNS void AS $$
+DECLARE  
+	class          jsonb;
+	obj_id 		   uuid;
+	value_to_add   jsonb; 
+	json_path      text[];
+	obj		 	   jsonb;
+	new_obj		   jsonb;
+	field_value    jsonb;
+	access_token   jsonb;
```

Comment:
Spaces and tabs should not be mixed, otherwise the code looks ugly with different tab sizes and, therefore, less readable. I would recommend to uses spaces only in order to ensure the same look of the code in different viewers.

Reply:
thanks, done